src/wowman/core.clj

```diff
@@ -87,13 +93,23 @@
 
    ;; final config, result of merging :file-opts and :cli-opts
    :cfg {:addon-dir-list []
-         :debug? false}
+         :debug? false ;; todo, remove
+         :selected-catalog :short}
+
+   :catalog-source-list [{:name :short :label "Short (default)" :source "https://raw.githubusercontent.com/ogri-la/wowman-data/master/catalog-short.json"}
+                         ;; todo: change the name of the release from 'daily' to something else
+                         {:name :full :label "Full" :source remote-catalog}
```

Comment:
inline value for `remote-catalog`

can we make the paths consistent? we have three new catalogs to upload as part of the release, should we even both with releases? can we just push an update from Travis? investigate in another PR